src/plugins/feature_catalogue/README.md

```diff
@@ -0,0 +1,5 @@
+# Feature catalogue plugin
+
+Replaces the legacy `ui/registry/feature_catalogue` module for registering "features" that should be showed in the home
+page's feature catalogue. This should not be confused with the "feature" plugin for registering features used to derive
```

Comment:
then probably we should rename either?

Reply:
I think this will be more clear once this registry moves into a "home" plugin with the actual UI. I opted not to name this plugin "home" right now since it may not be split out until 8.x and I thought that would be confusing in the meantime.